src/lowertest/README.md

```diff
@@ -87,8 +95,20 @@ recommended that the object also implement `serde::Serialize`.
 In order to translate the text syntax into a correctly-deserializable JSON
 string, we need information about enums and structs being created that are not
 specified in the test syntax.
-Thus, you need to derive the trait `MzEnumReflect` (resp. `MzStructReflect`) for
-each enum (resp. struct) that you will need to construct your object.
+
+Thus, you need to derive the trait `MzReflect` for each enum or struct that
+you will need to construct your object. If you cannot derive the enum or struct
+because it is from an external crate, then you should add its name to
+`EXTERNAL_TYPES` in [../lowertest-derive/src/lib.rs](../lowertest-derive/src/lib.rs).
+
+You can add the attribute `#[mzreflect(ignore)]` to a field that you don't want
+to derive the trait `MzReflect` for. If you do this, you agree to be
```

Comment:
Can we add a sentence intimating why you wouldn't want to derive `MzReflect`, even at the highest level?

Reply:
I thought about it, and it should be perfectly fine to derive `MzReflect` anywhere. It's just that you may not need to derive `MzReflect` if you have implemented an alternate syntax for constructing the enum variant or struct. Thus, you may want to communicate to `lowertest` that when constructing your test object, you do not need the particular type to be in `ReflectedTypeInfo`.